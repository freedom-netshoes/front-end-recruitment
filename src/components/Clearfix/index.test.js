import Clearfix from './';

describe('Clearfix', () => {
  it('should render correctly', () => {
    const wrapper = shallow(
      <Clearfix />
    );

    expect(wrapper).toMatchSnapshot();
  });
});
