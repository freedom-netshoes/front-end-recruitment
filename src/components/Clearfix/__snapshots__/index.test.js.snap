// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Clearfix should render correctly 1`] = `
<div
  className="clearfix"
/>
`;
